kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -16,6 +194,183 @@
 
   export default {
     name: 'SectionEditor',
+    components: {},
+    data() {
+      return {
+        selectedQuestionOrder: '',
+        numQuestions: 1,
+      };
+    },
+    $trs: {
```

Comment:
I have two high-level comments about strings here. 

First, rather than put strings in the component and use strings in the component, we can now use the strings in `enhancedQuizManagementStrings` (see kolibri-common/strings/enhancedQuizManagementStrings.js). Based on the team feedback, we are going to try this out for this project. I know we were a bit delayed in getting these in, but I think having just one place for strings will hopefully make things easier. I think it is worth refactoring the strings here now, rather than merging these in and having to do the cleanup later. @nucleogenesis can probably do a cohack session with you (and @thanksameeelian if she is curious - I know she has some frontend work coming up) about how we will be accessing and using these strings within the components. Hopefully it will be easier. And, any edits that you need to make to the strings can be made directly to that file. 

Secondly, and relatedly, I understand that you are just scaffolding out the UI here, but I think we should try to refrain from adding placeholder strings (such as "2 resources selected from 2 channels") unless they are absolutely necessary. I realize that the scope of this PR might not include all of the functionality that would enable some of the interactions on this section of the page, but I would prefer that use the real counts (i.e. if we don't have any selected resources, just have it be the data which would be 0) rather than pass in placeholder strings or placeholder values. I'm not sure I'm explaining this very well, so please do ask if you have questions about what I mean.